oteapi/strategies/parse/text_csv.py

```diff
@@ -233,10 +234,9 @@ class CSVConfig(AttrDict):
 class CSVResourceConfig(ResourceConfig):
     """CSV parse strategy filter config."""
 
-    mediaType: str = Field(
+    mediaType: Literal["text/csv"] = Field(
         "text/csv",
```

Comment:
Consider comment: "text/csv", # this should match the string in the Literal above

Reply:
I don't like the way you have to write out "text/csv" twice, sounds like you need to do this, but it seems weird

Reply:
The type explains that the value can _only_ be `"text/csv"`, which means validation will fail if it is anything else. This is the behavior we want for the CSVStrategy.
The second `"text/csv"` is the default value that will be set for `mediaType` should it not be explicitly given.

One could argue that `mediaType` must ALWAYS be explicitly provided, and hence the default value can be changed to an ellipsis (`...`) to signify this? I don't mind that change at all. This will reflect what is originally intended for the `ResourceConfig` as well, but can only be realized through a custom validator due to it only being valid if different pairs of fields are given at the same time (`mediaType` + `downloadUrl` OR `accessUrl` + `accessService`).

Reply:
Yeah a bit hard to say, might need a separate issue?

Reply:
Opened #359 to follow up on this.